services/gitlab/gitlab-coverage.service.js

```diff
@@ -0,0 +1,101 @@
+'use strict'
+
+const Joi = require('@hapi/joi')
+const { coveragePercentage } = require('../color-formatters')
+const { optionalUrl } = require('../validators')
+const { BaseSvgScrapingService, NotFound } = require('..')
+
+const badgeSchema = Joi.object({
+  message: Joi.alternatives()
+    .try(Joi.string().regex(/^([0-9]+\.[0-9]+%)|unknown$/))
+    .required(),
```

Comment:
No need for `alternatives` now
```suggestion
  message: Joi.string().regex(/^([0-9]+\.[0-9]+%)|unknown$/).required(),
```